lib/core/renderer.js

```diff
@@ -341,8 +341,18 @@ export default class Renderer {
       return { html, getPreloadFiles }
     }
 
+    if (this.noSSR || spa) {
+      return spaRenderer()
+    }
+
     // Call renderToString from the bundleRenderer and generate the HTML (will update the context as well)
-    let APP = await this.bundleRenderer.renderToString(context)
+    let APP
+    try {
+      APP = await this.bundleRenderer.renderToString(context)
+    } catch (_) {
+      consola.warn(`ssrRenderer fails, fall back to spaRenderer for path ${url}.`)
```

Comment:
I consider this a breaking change. Because Nuxt behaviour regarding SSR errors will be completely different